components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go

```diff
@@ -96,29 +98,34 @@ func (p *ProvisionAzureEventHubStep) Run(operation internal.ProvisioningOperatio
 		azure.TagOperationID:  &operation.ID,
 	}
 
+	// Generating a unique name for resource group and namespace with operation ID as suffix if available
+	uniqueName, err := p.getUniqueNSName(operation.ID, azureClient, log)
```

Comment:
thinking ... should we use `instance.id` instead? I am not sure about the relation of operation to instance. Does an instance always have only one operation?